WordPress/gutenberg
# Decode HTML entities in success notices on the Pages screen

## What?

Decodes HTML entities in the post and page titles in the success notices that shows when creating a new page through the Site Editor, and when duplicating a post or page via the Site Editor Pages screen or the block editor.

Note: This meant adding "@wordpress/html-entities" to the `edit-post` package. 

Partial for https://github.com/WordPress/gutenberg/issues/60995

## Why?

When a page or post title has HTML entities like `&` in the post title, these success notices displayed the characters as encoded. So `&` displayed as `&#038`;.

## How?

Wrap the titles inside `decodeEntities` https://developer.wordpress.org/block-editor/reference-guides/packages/packages-html-entities/#decodeentities

## Testing Instructions

- Go to Appearance > Editor > Pages.
- Create a new Page, add a & to the title, or some other HTML entity you want to test.
- Because your new page is blank, the pattern selection modal may pop up: you may need to close it to see the snackbar notice in the bottom corner.
- Confirm that all characters in the notice display correctly, and are not encoded.

----

- Locate the Actions menu in the Page panel.
- Select "Duplicate" in the drop down.
- Do not change the title, keep the character you added in the previous step. Duplicate.
- Confirm that all characters in the notice display correctly, and are not encoded.

----

- Go to the Pages screen in the Site Editor ( Appearance > Editor > Pages).
- Select View options, Layout:  Grid or Table.
- Locate your page with the HTML entity in the title. Open the Actions menu (the three dots). 
- Select "Duplicate".
- Do not change the title, keep the character you added in the previous step. Duplicate.
- Confirm that all characters in the notice display correctly, and are not encoded.

----

- Go back one step so that the Site Editor page detail screen shows ( the dark left hand sidebar with information about the individual page).
- Locate the actions menu and select "Duplicate".
- Do not change the title, keep the character you added in the previous step. Duplicate.
- Confirm that all characters in the notice display correctly, and are not encoded.

## Screenshots or screencast
